plugins/woocommerce/package.json

```
@@ -48,6 +50,7 @@
 		"test:api": "API_TEST_REPORT_DIR=\"$PWD/tests/api\" pnpm exec wc-api-tests test api",
 		"test:api-pw": "USE_WP_ENV=1 pnpm playwright test --config=tests/api-core-tests/playwright.config.js",
 		"test:e2e-pw": "USE_WP_ENV=1 pnpm playwright test --config=tests/e2e-pw/playwright.config.js",
+		"test:perf": "./k6 run ./tests/performance/tests/gh-action-pr-requests.js",
```

`./k6` doesn't exist
```suggestion
		"test:perf": "k6 run ./tests/performance/tests/gh-action-pr-requests.js",
```

---

It should exist after the prepare test environment step (`pnpm env:perf`) installs it. Changing this will result in `sh: 1: k6: not found` error, see [this run](https://github.com/woocommerce/woocommerce/actions/runs/8355851992/job/22871783691?pr=45190#step:5:17).